adafruit_max7219/max7219.py

```diff
@@ -96,37 +111,42 @@ def brightness(self, value):
             raise ValueError("Brightness out of range")
         self.write_cmd(_INTENSITY, value)
 
-    def show(self):
+    def show(self) -> None:
         """
         Updates the display.
         """
         for ypos in range(8):
             self.write_cmd(_DIGIT0 + ypos, self._buffer[ypos])
 
-    def fill(self, bit_value):
+    def fill(self, bit_value: int) -> None:
         """
         Fill the display buffer.
 
         :param int bit_value: value > 0 set the buffer bit, else clears the buffer bit
         """
         self.framebuf.fill(bit_value)
 
-    def pixel(self, xpos, ypos, bit_value=None):
+    def pixel(self, xpos: int, ypos: int, bit_value: int = None) -> None:
         """
         Set one buffer bit
 
-        :param xpos: x position to set bit
-        :param ypos: y position to set bit
+        :param int xpos: x position to set bit
+        :param int ypos: y position to set bit
         :param int bit_value: value > 0 sets the buffer bit, else clears the buffer bit
         """
         bit_value = 0x01 if bit_value else 0x00
         self.framebuf.pixel(xpos, ypos, bit_value)
 
-    def scroll(self, delta_x, delta_y):
-        """Srcolls the display using delta_x,delta_y."""
+    def scroll(self, delta_x: int, delta_y: int) -> None:
+        """
+        Srcolls the display using delta_x,delta_y.
+
+        :param int delta_x: positions to scroll in the x direction
+        :param int delta_y: positions to scroll in the y direction
+        """
         self.framebuf.scroll(delta_x, delta_y)
 
-    def write_cmd(self, cmd, data):
+    def write_cmd(self, cmd, data) -> None:
```

Comment:
The arguments for `write_cmd` are missing their types. grepping for usages it seems to me these are both `int`

Can you add them here.
```
def write_cmd(self, cmd: int, data: int) -> None:
```

Reply:
I've updated the types for `write_cmd` as described.

I also took a look at the [datasheet](https://cdn-shop.adafruit.com/datasheets/MAX7219.pdf) to make the docstring more useful.  In this case Table 1 shows that the first byte of the `bytearray`, `cmd`, corresponds to the register address while the second byte, `data`, corresponds to the data written to that register.  I have a cheap Amazon MAX7219 LED matrix, and observing the calls to `write_cmd` and their corresponding impact on the display confirm that the values are of type `int` and that the datasheet is accurate.